release_notes/ose_3_2_release_notes.adoc

```diff
@@ -764,12 +763,168 @@ OpenShift Enterprise entitlements for OpenShift Enterprise errata notification
+The installer uses the `repoquery` command, which is provided by the *yum-utils*
+package and is not in Minimal installations of Red Hat Enterprise Linux 7.x.
+Ansible 1.9 installed this package before calling the command, but it is no
+longer installed starting with Ansible 2.0. This bug fix updates the installer
+to check that the *yum-utils* package is installed, and attempts to install it
+if it is not.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1334639[BZ#1334639]::
+When configuring Ansible variables in inventories using raw booleans,
+installations could fail due to broken master configurations. This bug fix
+updates the installer to ensure that these values are properly converted to the
+master configuration files.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1334148[BZ#1334148]::
+The default for the `*openshift_docker_hosted_registry_insecure*` Ansible
+variable is `true` but if it was set explicitly to `true` in an inventory, the
+installation would product an error. Setting the variable to `false` caused it
+to be ignored. This bug fix updates the installer to respect explicitly setting
+this value.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1329496[BZ#1329496]::
+Previously, the `*osm_default_subdomain*` Ansible variable did not take effect when
+set. This was due to a backwards compatibility issue in the installer. This bug
+fix updates the installer to once again respect setting this variable.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1326045[BZ#1326045]::
+The legacy `*cli_docker_options*` and `*cli_docker_log_options*` Ansible
+variables were not working due to use of an outdated host group that was since
+refactored. The variables were supposed to be migrated to the new format, using
+the `*openshift_docker_options*` and `*openshift_docker_log_options*` variables,
+respectively. This bug fix updates the installer so that the legacy variables
+can be used again.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1326642[BZ#1326642]::
+During an upgrade, if the `*openshift_image_tag*` Ansible variable was set in an
+inventory to an image version that was older than the latest available, the
+latest available version was still set in the *systemd* unit files. This bug fix
+updates the installer to ensure the version set by `*openshift_image_tag*` is
+what actually gets set in the *systemd* unit files.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1336202[BZ#1336202]::
+Upgrades from OpenShift Enterprise (OSE) 3.1 to 3.2 on RPM-based installations
+incorrectly attempted to pull the *openshift3/ose:latest* image. This step is
+only required for containerized installations and has been removed from
+RPM-based installations, eliminating the need to pull an unexpected image.
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1331389[BZ#1331389]::
+Previously, the `*cli_docker_additional_registries*` Ansible variable did not
+take effect during an upgrade. This was due to legacy options (`*cli_**`) not
```

Comment:
Is the double asterisk on cli_ correct? I'm assuming so, but noting it just in case.